Game2048Intelligence/src/main/java/heuristic/SnakeHeuristic.java

```diff
@@ -0,0 +1,59 @@
+//package heuristic;
```

Comment:
Might be a good idea to avoid checking in commented code. You could save it outside of the project if it's something still in progress.

Reply:
I'll ask you how to do that.